Backend/Backend/Controllers/Requests/EpisodeRequest.cs

```diff
@@ -49,4 +49,24 @@ public class EditEpisodeRequest
     }
 
 
+    [BindProperties]
+    public class GetEpisodeResponse {
```

Comment:
is the naming here correct? I think this is a request, not a response right?

Reply:
this is the Response object we send